src/lidar/bg_flood_model.py

```diff
@@ -109,40 +112,44 @@ class BGDEM(Base):
 
 
 def run_model(
-    bg_path,
-    instructions,
-    catchment_boundary,
-    resolution,
-    endtime,
-    outputtimestep,
-    engine,
+        bg_path,
+        instructions,
+        catchment_boundary,
+        resolution,
+        endtime,
+        outputtimestep,
+        engine,
 ):
     """Call the functions."""
     dem_path = dem_metadata_in_db.get_dem_path(instructions, engine)
     bg_model_inputs(
         bg_path, dem_path, catchment_boundary, resolution, endtime, outputtimestep
     )
     os.chdir(rf"{bg_path}")
-    subprocess.call([rf"{bg_path}\BG_Flood_Cleanup.exe"])
+    subprocess.call([rf"{bg_path}/BG_Flood_Cleanup.exe"])
+
+
+def get_api_key(key_name: str):
+    """Get the required api key from dotenv environment variable file"""
+    env_path = pathlib.Path().cwd() / pathlib.Path('src/.env')
+    load_dotenv(env_path)
+    api_key = os.getenv(key_name)
+    return api_key
 
 
 def main():
     engine = setup_environment.get_database()
-    bg_path = r"P:\DT\BG-Flood\BG-Flood_Win10_v0.6-a"
+    bg_path = r"U:/Research/FloodRiskResearch/DigitalTwin/BG-Flood/BG-Flood_Win10_v0.6-a"
```

Comment:
same as above RE using pathlib in future PRs

Reply:
done